src/components/forms/sendEmailForm/index.tsx

```diff
@@ -98,6 +102,11 @@ const SendEmailForm: React.FC<SendEmailFormProps> = ({ emails }) => {
         rules={requiredRule(t('body_required'))}
         hidden={showPreview}
       >
+        <Input.TextArea
+          name="emailContent"
+          rows={6}
+          placeholder={'Email Body'}
+        />
         <Input.TextArea rows={8} placeholder={t('body_placeholder')} />
```

Comment:
⛏️ Can we also remove this extra `TextArea`? There's two text areas here:

![image](https://github.com/Code-4-Community/speak-for-the-trees-frontend/assets/68914997/055dc6d9-03b1-4a44-82b7-2032c77d60dd)